pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/LocalRunner.java

```diff
@@ -120,39 +120,35 @@ void start() throws Exception {
             if (builtInSource != null) {
                 sourceConfig.setArchive(builtInSource);
             }
-            NarClassLoader classLoader;
```

Comment:
does this mean we cannot run a built-in source or sink locally?

Reply:
Built-ins should be passed to the localrun with a proper file location, so they can be